include/cucumber-cpp/internal/drivers/QtTestDriver.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef CUKE_QTTESTDRIVER_HPP_
+#define CUKE_QTTESTDRIVER_HPP_
+
+#include "../step/StepManager.hpp"
+
+namespace cucumber {
+namespace internal {
+
+class QtTestStep : public BasicStep{
+public:
+    QtTestStep(): IsInitialized(false), IsCleanedUp(false), TestRun(false) {}
```

Comment:
Is there a reason why here and in several other places you are using UpperCamelCase instead of lowerCamelCase?